docs/src/components/motion/easing.module.css

```diff
@@ -0,0 +1,166 @@
+b {
```

Comment:
haven't seen a `b` in a while. why use this instead of `a strong`?

Reply:
no reason.  it's been there for years.